components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/impl/OAuthOpaqueAuthenticatorImpl.java

```diff
@@ -278,8 +278,8 @@ public OrganizationInfo getOrganizationInfo(String username) throws APIManagemen
         String organization = null;
         String organizationId = null;
         try {
-            if (tenantDomain.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
```

Comment:
need to do other way round

Reply:
`tenantDomain` is not returned as null in `MultitenantUtils.getTenantDomain(username)` method. but changed the order